plugin-server/src/main/ingestion-queues/session-recording/services/console-logs-ingester.ts

```diff
@@ -163,6 +163,7 @@ export class ConsoleLogsIngester {
                     topic: KAFKA_LOG_ENTRIES,
                     value: Buffer.from(JSON.stringify(cle)),
                     key: event.session_id,
+                    waitForAck: true,
```

Comment:
These recordings calls were calling `produce` directly which defaulted to `true`, so this isn't a functionality change.

But maybe we don't actually need/want to wait? @pauldambra